packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -45,11 +53,11 @@ export async function processOutboundPatientDiscoveryResps({
       results
     );
 
-    await processPatientDiscoveryProgress({ patient: patientIds, status: "completed" });
-
     const patient = await getPatientOrFail({ id: patientId, cxId });
-    const startedAt = patient.data.patientDiscovery?.startedAt;
+    const cqData = getCQData(patient.data.externalData);
+    const startedAt = cqData?.discoveryStartedAt;
 
+    // BUG There's no gurantee the requestId of the current PD matches this request ID
```

Comment:
Why is this the case? For every requestId dont we go through the whole flow?

Reply:
The bug is actually with `startedAt` -- removing the comment as generally the state issue is flagged in a bunch of places and we're better off having a wholistic issue or something.

Reply:
Done.

Reply:
Larger state handling issue https://github.com/metriport/metriport-internal/issues/1832 (put a TODO in the code for each place right now we enter the PD flow outside of the scheduling framework).